### PR TITLE
Change withdrawal amount unmarshal to uint64 (gwei)

### PR DESCRIPTION
**Pull request by terencechain**
This PR implements https://github.com/ethereum/execution-apis/pull/354. It updates the unmarshal for Engine API with the change to use the same units (Gwei) for the amount in each withdrawal as the consensus layer

### CODE REVIEW

**Comment by terencechain** [comment]
Do not merge this until #354 is merged

**Comment by rkapka** [comment]
> Do not merge this until #354 is merged

#354 is some very old Prysm PR

**Review by potuz — APPROVED** [review]

**Pull request merged**
